packages/patternfly-4/src/content/design-guidelines/usage-and-behavior/empty-state/empty-state.md

```diff
@@ -5,64 +5,186 @@ hideTOC: true
+
+It is OK to use “you” and “your” but never assign blame to the user.
+
+![Accusatory](./img/accusatory.jpg)
+
+There is also no need to be overly mannerly, so avoid using please, unless the user is being inconvenienced. 
+
+![Please](./img/please.jpg)
+
+**Level of detail**
+
+In your empty state, state what isn't there, and then give the user a next step.  Empty states should not be used to describe in detail how to use the page—the design should be intuitive enough without extensive written explanation. Include at least one sentence, even if it seems like a straightforward explanation, and at least two when there may be more explanation needed.
+
+![Detail](./img/level-of-detail.jpg)
+
+**Sentence case**
+
+Empty states should follow sentence case rules. Only the first word in any phrase or sentence should be capitalized (whether it is in the body or in a button or link), unless a proper noun is included in the phrase/sentence.
+Every sentence in a body should end with a period. Headers should not end in a period though.
+
+![Sentence case](./img/sentence-case.jpg)
+
+### Primary and secondary buttons
+Buttons in empty states are used to highlight next steps a user can take when they reach your page. There are multiple ways to set up calls-to-action. You can have a call-to-action presented as a primary button, or within the body. If you have a singular call-to-action on your page, it is preferable for it to be presented in primary button format as it places bigger emphasis on the call-to-action. However, there may be a specific reason why you may need it embedded in the body.
+
+1. **Call-to-action as a primary button**
+![Primary button one cta](./img/one-ctas-primary.jpg) 
+
+2. **Call-to-action in body**
+![In body cta](./img/one-cta-body.jpg)
+
+If you have more than one actionable link, there are two options depending on how your primary call-to-action is presented:
+1. **Link button underneath the primary button:** if your primary call-to-action was in a primary button format
+
+![Primary button two ctas](./img/two-ctas-primary.jpg) 
+
+2. **Link button underneath the body:** if your first call to action was in the body.
+
+![In body two ctas](./img/two-ctas-body.jpg)
+
+There are only two situations where a stand-alone link button (with nothing linked in the body) can be used:
+1. **Filtering or search situation:** when a user has filtered for something in a table, and is prompted to "clear all filters" or "include" other items in their search
+2. **Multiple primary calls-to-action:** when there are multiple primary actions the user can take and they are all equally important
```

Comment:
Is there an image missing here? Seems like the example below relates to filtering or search situations, but I don't see an example that illustrates this (multiple primary calls to action)

Reply:
As discussed, there is only an example available for the filtering situation, so I moved the screenshot up to be underneath the first bullet!